node/app.js

```diff
@@ -0,0 +1,60 @@
+const HTTP = require('http')
+const { $fetch, encrypt, decrypt, $query } = require('./tool')
+const PORT = 1954
+const { SERVER } = process.env
+const $remote = SERVER === 'celwk'
+const domain = $remote ? 'https://wcraft.canskit.com' : 'https://canskit.wcraft.cn' // Antoher Lisp Version: https://wcraft.celwk.com
+HTTP.createServer(async (req, res) => {
+    const { headers, url, method } = req
+    console.log(headers)
+    // console.log({ headers, url, method })
+
+    res.writeHead(200, {'Content-Type': 'application/json'})
```

Comment:
does it mean for any request the HTTP response code would be 200?

Reply:
For testing purpose, I haven't paid attention to authentication yet.(That's not for this project specially?)